static/wix.xml

```diff
@@ -72,10 +76,28 @@
       </Component>
     </DirectoryRef>
 
-    <!-- Step 4: Tell WiX to install the files -->
+   <!-- Step 4: Add app desktop shortcut --> 
+    <DirectoryRef Id="DesktopFolder">
+      <Component Id="DesktopShortcut" Guid="{{DesktopShortcutGuid}}" >
+          <Shortcut Id="MyDesktopShortcut"
+                    Name="{{ShortcutName}}"
+                    Description="{{ApplicationDescription}}"
+                    Target="[APPLICATIONROOTDIRECTORY]{{ApplicationBinary}}.exe"
```

Comment:
This seems point to the installer... Instead of the acturial app exe. See the new folder structure.